docs/src/content/docs/es/guides/authoring-content.md

```diff
@@ -201,9 +201,148 @@ var fun = function lang(l) {
-```
+### Características de Expressive Code
+
+Starlight usa [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) para ampliar las posibilidades de formato de los bloques de código.
+Los marcadores de texto de Expressive Code y los plugins de marcos de ventana están habilitados de forma predeterminada.
+El renderizado de los bloques de código se puede configurar utilizando la opción de configuración [`expressiveCode`](/es/reference/configuration/#expressivecode) de Starlight.
+
+#### Marcadores de texto
+
+Puedes resaltar líneas específicas o partes de tus bloques de código utilizando [los marcadores de texto de Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#usage-in-markdown--mdx-documents) en la línea de apertura de tu bloque de código.
+Usa llaves (`{ }`) para resaltar líneas enteras, y comillas para resaltar cadenas de texto.
+
+Hay tres estilos de resaltado: neutral para llamar la atención sobre el código, verde para indicar código insertado y rojo para indicar código eliminado.
+Tanto el texto como las líneas enteras pueden marcarse con el marcador predeterminado, o en combinación con `ins=` y `del=` para producir el resaltado deseado.
+
+Expressive Code proporciona varias opciones para personalizar la apariencia visual de tus ejemplos de código.
+Muchas de estas opciones se pueden combinar, para obtener ejemplos de código altamente ilustrativos.
+Por favor, explora la [documentación de Expressive Code](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md) para ver las extensas opciones disponibles.
+Algunos de los ejemplos más comunes se muestran a continuación:
+
+- [Marca líneas enteras y rangos de líneas usando el marcador `{ }`](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#marking-entire-lines--line-ranges):
+
+  ```js {2-3}
+  function demo() {
+    // Esta línea (#2) y la siguiente están resaltadas
+    retrun 'Esta es la línea #3 de este fragmento'
+  }
+  ```
+
+  ````md
+  ```js {2-3}
+  function demo() {
+    // Esta línea (#2) y la siguiente están resaltadas
+    return 'Esta es la línea #3 de este fragmento'
+  }
+  ```
+  ````
+
+- [Marca selecciones de texto usando el marcador `" "` o expresiones regulares](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#marking-individual-text-inside-lines):
+
+  ```js "Términos individuales" /Even.*supported/
+  // Términos individuales también pueden ser resaltados
+  function demo() {
+    return 'También las expresiones regulares son compatibles'
+  }
+  ```
+
+  ````md
+  ```js "Términos individuales" /Even.*supported/
```

Comment:
I think the regular expression here needs to be updated to match the translated code:

`````suggestion
  ```js "Términos individuales" /También.*compatibles/
  // Términos individuales también pueden ser resaltados
  function demo() {
    return 'También las expresiones regulares son compatibles'
  }
  ```

  ````md
  ```js "Términos individuales" /También.*compatibles/
`````